daprdocs/content/en/operations/hosting/self-hosted/self-hosted-overview.md

```diff
@@ -25,3 +25,8 @@ The `dapr-placement` service is responsible for managing the actor distribution
 
 You can use the [`dapr run` CLI command]({{< ref dapr-run.md >}}) to a Dapr sidecar process along with your application.
 
+## Name resolution
+
+Dapr uses a [name resolution component]({{< ref supported-name-resolution >}}) for service discovery within the [service invocation]({{< ref service-invocation >}}) building block. By default Dapr uses mDNS when in self-hosted mode.
+
+If you are running Dapr on virtual machines or across multiple machines you can use [HashiCorp Consul]({{< ref setup-nr-consul.md >}}).
```

Comment:
I think this sentence should read something like this.... "If you are running Dapr on virtual machines or where mDNS is not available then use  [HashiCorp Consul]({{< ref setup-nr-consul.md >}}) component for name resolution."

Reply:
Perfect thanks! Also in future comments feel free to use CTRL + g to add a suggestion that I can commit